k8s/prod-analytics-frontend.yaml

```diff
@@ -0,0 +1,41 @@
+###########################################################
+# Start deployment of analytics frontend
```

Comment:
Small recommendation:
We could remove the analytics configs since it no longer exists.

Reply:
@Baalmart : Resolved